job-service-scheduled-executor/src/main/java/com/hpe/caf/services/job/scheduled/executor/QueueServicesFactory.java

```diff
@@ -102,8 +101,11 @@ public static QueueServices create(final String targetQueue, final String partit
         }
 
         //  Declare worker queue.
-        LOG.debug("Passively declaring worker queue {}...", stagingQueueOrTargetQueue);
-        publishChannel.queueDeclarePassive(stagingQueueOrTargetQueue);
+        LOG.debug("Declaring worker queue {}...", stagingQueueOrTargetQueue);
+        final Map<String, Object> queueArguments = new HashMap<>();
+        queueArguments.put("x-queue-type", "quorum");
```

Comment:
These need to come from configuration, much like how workers have their queues configured.
You have to handle classic queues with a max-priority and quorum without priority.

Reply:
Addressed